scripts/borg.exe.spec

```diff
@@ -33,33 +33,53 @@ a = Analysis([os.path.join(basepath, 'src', 'borg', '__main__.py'), ],
              win_private_assemblies=False,
              cipher=block_cipher)
 
+pyz = PYZ(a.pure, a.zipped_data, cipher=block_cipher)
+
+# For macOS we build a directory-based binary to get around repeated
+# lib validations. See borgbackup/borg#5185 and
```

Comment:
and ... ?

Reply:
And what? More intro text?

Reply:
I mean that line 39 ending with "and" - there seems to be something missing.

Reply:
looks like that change is not present any more in this PR, so it is resolved.